src/ripple/app/main/GRPCServer.cpp

```diff
@@ -427,9 +428,9 @@ GRPCServerImpl::GRPCServerImpl(Application& app)
     : app_(app), journal_(app_.journal("gRPC Server"))
 {
     // if present, get endpoint from config
-    if (app_.config().exists("port_grpc"))
+    if (app_.config().exists(SECTION_PORT_GRPC))
     {
-        const auto& section = app_.config().section("port_grpc");
+        Section section = app_.config().section(SECTION_PORT_GRPC);
```

Comment:
Don't make a copy of the `Section`. 
```
Section const& section = ...
```

Reply:
fixed in dc4a1beb066afbce6ed123923b1d134aea68e372